stylesheets/pages/products.css

```diff
@@ -61,3 +61,9 @@
   gap: var(--spacing-sm);
   padding: 0 0 var(--spacing-xs) 0;
 }
+@media only screen and (max-width:580px){
+  .page-main{
+    grid-template-columns: 1fr;
+    gap:var(--spacing-xl);
+  }
+}
```

Comment:
Clear and concise commit messages help the reviewer while reviewing. Keep up this good practice.

Reply:
sure